### DISCUSSION TITLE
Setup guide for virtual bones

### DISCUSSION THREAD

**Discussion by CanadianCheesus** [Sixze/ALS-Refactored#185] [Q&A]
I would like to add virtual bones to another skeleton, is there a guide for how to do this with this version of ALS?

**Comment by ameaninglessname** [comment]
[Have you checked the official document?](https://docs.unrealengine.com/5.1/en-US/virtual-bones-in-unreal-engine/)

**Reply by CanadianCheesus** [reply]
For some reason I can't load docs.unrealengine.com right now... 

I know how to add virtual bones and have done this process with other versions of ALSv4, I just want to make sure I am handling "ik_hand_r_no_gun_bone", "ik_hand_r_with_gun_bone", "ik_hand_l_no_gun_bone" and "ik_hand_l_with_gun_bone" correctly.
